apps/www/data/Developers.tsx

```diff
@@ -66,7 +66,7 @@ export const data = {
         {
           text: 'Events & Webinars',
           url: '/events',
-          icon: IconCalendar,
+          icon: Calendar,
```

Comment:
@ivasilov watch out, the calendar icon on the "Events and webinars" is an actual calendar 😁
You probably want the { Calendar } from lucide-react.

<img width="796" alt="Screenshot 2024-09-04 at 11 34 25" src="https://github.com/user-attachments/assets/3486caa9-91e5-4416-9762-4534cc4687c9">